template/package.json

```diff
@@ -9,8 +9,10 @@
         "build": "vertigis-web-sdk build",
         "start": "vertigis-web-sdk start"
     },
-    "dependencies": {},
+    "dependencies": {
+        "@vertigis/web-sdk": "file:.."
```

Comment:
I haven't seen this syntax before. Does this cause this version to match the file of the same name a directory up?

Reply:
Oops! Thanks for catching that. This was not supposed to be committed -- I was using it for testing. 

You can use the `file:` syntax to either point to a *.tgz archive of a package (usually produced by `npm pack`), or a folder. At some point I used this to get around the fact that npm install in a newly created SDK project was looking for "@vertigis/web-sdk" with version "0.0.0-semantically-released", which doesn't exist of course. I've removed it now.